html-forms/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Forms</title>
+  </head>
+  <body>
+
+    <h1>Coding Form!</h1>
+
+    <div>
+      <label for="email">Email</label>
+    </div>
+
+    <input type="text" name="email" />
+
+    <div>
+      Why is coding cool?
+    </div>
+
+    <select name="Select an Option">
```

Comment:
Give this `<select>` an id to connect to the `<label>` element you will create above ^